app/views/posts/index.html.erb

```diff
@@ -187,9 +189,12 @@
                     <label class="control-label col-lg-1" for="name">Pre-pay: </label>
                     <div class="col-lg-8">
                       <input id="prepay" class="checkbox-inline" type="checkbox" name="need_pre_pay" value="" ng-model="paymentSelected">
-                      <i class="fa fa-inr"></i>
-                      <input type="text" class="form-control pay-amount" placeholder="0.0" maxlength="8" ng-model="prepayAmount"> by
-                      <input type="text" class="form-control pay-amount-by" ng-model="prepayAmountBy">
+                      <span ng-show="paymentSelected">
+                        <i class="fa fa-inr prepay-symbol-green" ng-show="show_green"></i>
```

Comment:
Instead of having two different `<i>` tags please change the class , no need of using such complex logic